libse/Utilities.cs

```diff
@@ -468,17 +468,20 @@ public static string AutoBreakLinePrivate(string text, int maximumLength, int me
                 var noTagLines = HtmlUtil.RemoveHtmlTags(text, true).SplitToLines();
                 if (noTagLines.Count == 2)
                 {
-                    var arr0 = noTagLines[0].Trim().TrimEnd('"', '\'').TrimEnd();
+                    const string LineCloseChars = ".?!)]♪";
+                    char[] quoteChars = { '"', '\'' };
+
+                    var arr0 = noTagLines[0].Trim().Trim(quoteChars).Trim();

```

Comment:
Trim quotes chars from both side... the language is not checked yet so we don't know if it is going to be Arabic or not!